src/ui/public/partials/paginated_selectable_list.html

```diff
@@ -0,0 +1,51 @@
+<form role="form" class="form-inline">
+  <div class="container-fluid">
+    <div class="row">
+      <div class="input-group form-group finder-form col-md-9">
+        <span class="input-group-addon">
+          <i class="fa fa-search"></i>
+        </span>
+        <input
+          input-focus
+          ng-model="query"
+          placeholder="Filter..."
+          class="form-control"
+          name="query"
+          type="text"
+          autocomplete="off" />
+      </div>
+      <div class="finder-hit-count col-md-3">
+        <span>{{filterCount}} of {{hitCount}}</span>
+      </div>
+    </div>
+  </div>
+</form>
+<paginate list="hits" per-page="{{perPage}}">
+  <ul class="li-striped list-group list-group-menu">
+    <li class="list-group-item list-group-menu-item">
+      <span class="paginate-heading">
+        Name
+        <i
+          class="fa"
+          ng-click="sortHits(hits)"
+          ng-class="isAscending ? 'fa-caret-up' : 'fa-caret-down'">
+        </i>
+      </span>
+    </li>
+    <li
```

Comment:
The entire `<li>` should be clickable here, not just the text. Take a look at how the saved-object-finder does it.